<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGen</class>
 <widget class="QWidget" name="KGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Key generator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="dialog-password">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Organization</string>
        </property>
        <property name="buddy">
         <cstring>lineEditOrganization</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditOrganization"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEditName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>E-mail</string>
        </property>
        <property name="buddy">
         <cstring>lineEditEmail</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Serial number</string>
        </property>
        <property name="buddy">
         <cstring>lineEditSerial</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditSerial">
        <property name="inputMask">
         <string>&gt;HHHHH-HHHHH-HHHHH-HHHHH-HHHHH; </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>134</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-close">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>134</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditOrganization</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditSerial</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditOrganization</sender>
   <signal>textEdited(QString)</signal>
   <receiver>KGen</receiver>
   <slot>changeData(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>KGen</receiver>
   <slot>changeData(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEmail</sender>
   <signal>textEdited(QString)</signal>
   <receiver>KGen</receiver>
   <slot>changeData(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>KGen</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeData(QString)</slot>
 </slots>
</ui>
